service/src/main/java/bio/terra/axonserver/service/cromwellworkflow/CromwellWorkflowService.java

```diff
@@ -134,6 +160,30 @@ public CromwellApiLabelsResponse getLabels(UUID workflowId)
         .labels(CROMWELL_CLIENT_API_VERSION, workflowId.toString());
   }
 
+  public Map<String, String> parseInputs(UUID workspaceId, String gcsPath, BearerToken token)
+      throws IOException, InvalidWdlException {
+    // 1) Get the WDL file and write it to disk
+    InputStream resourceObjectStream = fileService.getFile(token, workspaceId, gcsPath, null);
+    File targetFile = createSafeTempFile(UUID.randomUUID().toString(), "wdl");
```

Comment:
Perhaps for a separate PR, but we should delete the temp file after we're done with it. My suggestion is to refactor `createSafeTempFile` as an `Autocloseable` so that we can initialize the file in a try-with-resources block and the file will be automatically be deleted. It would be worth flagging that as a TODO here though.

Reply:
Just an FYI willy I'm working on this Autocloseable class in a separate PR, we can correct this ltaer

Reply:
Done, now using AutoClosable.